app/styles/main.scss

```diff
@@ -4,3 +4,7 @@
 
 @import 'containers/containers';
 @import 'components/components';
+
+body{
+  background:red;
```

Comment:
It would be better if we don't have variables with name "red". Instead it should be "background-color" or something descriptive.